fixtures/custom-rules/lib/testing/tfplan.rego

```diff
@@ -0,0 +1,8 @@
+package lib.testing
+
+import data.lib
+
+parse_fixture_file(fixture_file) = fixture {
```

Comment:
Expanding the existing `parse_fixture_file` functionality without modifying the existing templated `lib/testing/main.rego` file